pylib/openrazer/client/fx.py

```diff
@@ -717,6 +717,16 @@ def static(self, red: int, green: int, blue: int) -> bool:
             return True
         return False
 
+    def wave(self, direction: int) -> bool:
```

Comment:
Please make the pylib changes a separate PR

Reply:
Reverted the changes.  I'll submit a separate PR.